packages/Checkbox/src/Checkbox.styles.js

```diff
@@ -75,7 +75,7 @@ const styles = {
   },
 };
 
-const checkboxStyles = css`
+export const checkboxStyles = styled.div`
```

Comment:
Since this is no longer just the styles, it's an actual component, I'd recommend we change the naming convention (this comment applies to all the components in this PR).  In this case, probably `CheckboxWrapper` (Note the capital `C`), since it's not the _actual_ checkbox.

Reply:
Will make sure to adjust all of the names where needed